wiki/Version-4.0-User-Upgrade-Guide.md

```diff
@@ -352,7 +352,7 @@ Previously a ruleset could already "extend" an array property for a sniff set by
 
 As of PHP_CodeSniffer 4.0, a ruleset can also "extend" the default value of an array property as set in the sniff itself.
 
-The upside of this is, that if you want to default value + some extras, you no longer need to duplicate the default values from sniff array properties in your ruleset.  
+The upside of this is that if you want the default value + some extras, you no longer need to duplicate the default value from a sniff array property in your ruleset.  
```

Comment:
I believe I intended to write "want to use the default value + ..."

As for the change in the second part of the sentence... I still think plural correct here.

Reply:
> I believe I intended to write "want to use the default value + ..."

So something like the sentence below?

"The upside of this is that, if you want to use the default value + some extras, you no longer need to duplicate the default values from sniff array properties in your ruleset."

> As for the change in the second part of the sentence... I still think plural correct here.

I'm not sure about this one as well. In the version that I suggested above, plural seems correct to me as well.

Reply:
The proposed sentence looks good to me.

Reply:
Ok, I updated the sentence with the proposal we discussed.